apps/server/lib/google-drive/src/internal/shim-types.d.ts

```
@@ -0,0 +1,29 @@
+ *
+ * Note: these only exist at the type-level, there is no corresponding runtime
+ * version for any of these symbols.
+ */
+
+/**
+ * In order to properly access the global `NodeJS` type, if it's available, we
+ * need to make use of declaration shadowing. Without this, any checks for the
+ * presence of `NodeJS.ReadableStream` will fail.
+ */
+declare namespace NodeJS {
+	interface ReadableStream {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)